desktop_server/src/com/mygdx/game/server/network/handlers/CloseHandler.java

```diff
@@ -0,0 +1,29 @@
+package com.mygdx.game.server.network.handlers;
```

Comment:
Może jakiś wspólny interfejs dla handlerów?

Reply:
w sumie to nie widzę tego jeszcze, mam też trochę płonną nadzieję, że przejdziemy na jakiegoś protobuffa, albo serializację jsona